### ISSUE TITLE
US01-F01-SP02: システム上の補足説明文 

### ISSUE THREAD

**Issue by yaginuuu** [ViBii/mitemiru#305]

**Comment by Altairzym** [comment]
@sjc1209  このチケットは範囲として、「メトリクスを見る」画面だけ？それともシステム全体の説明文？

**Comment by sjc1209** [comment]
そうですね。説明が必要な部分は追加してもいいと思うので、アンケートの結果を見ながらどこが必要かわかるかもしれない。DONEの定義が書いたら、みなにレビューしてもらいましょう。もし補足があれば補足します。

**Comment by Altairzym** [comment]
@sjc1209 今どこ？！

**Comment by sjc1209** [comment]
図書館ですが。

**Comment by Altairzym** [comment]
@sjc1209  @ryoki1118 @yaginuuu アンケートの結果を見ながら、この様な提案をした、説明文の内容を指摘してください。

**Comment by ryoki1118** [comment]
>コミットグラフで赤線部分に単位（回数）を追加する
棒グラフ横の文を「コミット数(回)」にすればいいよ

**Comment by ryoki1118** [comment]
>タイトル：開発者全員生産性一覧

「生産性」は一切使わないようにしましょう。代わりに「効率」かな。
あとタイトルは日本語らしくないね。というかグラフにタイトルっているかね？

**Comment by Altairzym** [comment]
@ryoki1118 そうですね、「効率」のほうがいいと思う
